Source/Editor/Content/Proxy/CollisionDataProxy.cs

```diff
@@ -141,7 +141,13 @@ public void CreateCollisionDataFromModel(Model model, Action<CollisionData> crea
                 });
             };
             var initialName = (modelItem?.ShortName ?? Path.GetFileNameWithoutExtension(model.Path)) + " Collision";
-            Editor.Instance.Windows.ContentWin.NewItem(this, null, create, initialName, withRenaming);
+            
+            // If folder can not have assets then move to folder with the model and create asset
+            var contentWin = Editor.Instance.Windows.ContentWin;
+            if (!contentWin.CurrentViewFolder.CanHaveAssets)
+                contentWin.Navigate(modelItem?.ParentFolder?.Node);
```

Comment:
I think `ContentWindow.NewItem` could call `proxy.CanCreate` over that folder to check if it's a valid location, and then attempt to change it to Content so we have a more generic solution.

Reply:
Changed to now handling in `ContentWindow.NewItem`